Public/Src/Engine/Dll/EngineSchedule.cs

```diff
@@ -744,6 +744,12 @@ private bool ShouldSerializeOptimizationDataStructurePostExecution()
             IEnumerable<string> nonScrubbablePaths,
             ITempDirectoryCleaner tempCleaner)
         {
+            if (mountPathExpander != null)

```

Comment:
>if (mountPathExpander != null) [](start = 12, length = 30)

Could you add an explanation of the motivation/reason for this change (like the one you mentioned in your other comment) to the PR description and the commit message when you eventually commit?

It's a nice to have thing in commits/PRs in general so we can understand why changes were made in the future.

Reply:
Added context and the fix. Please let me know if you have any further comments.